src/Analyzers/Core/Analyzers/RemoveRedundantEquality/AbstractRemoveRedundantEqualityDiagnosticAnalyzer.cs

```diff
@@ -60,7 +64,7 @@ private void AnalyzeBinaryOperator(OperationAnalysisContext context)
 
             var isOperatorEquals = operation.OperatorKind == BinaryOperatorKind.Equals;
             _syntaxFacts.GetPartsOfBinaryExpression(operation.Syntax, out _, out var operatorToken, out _);
-            var properties = ImmutableDictionary.CreateBuilder<string, string>();
+            var properties = ImmutableDictionary.CreateBuilder<string, string?>();
```

Comment:
The dictionary doesn't hold any nulls. Both the keys and values are string constants defined in `RedundantEqualityConstants`. Am I missing something?

Reply:
I nullable enabled the file (which didn't help because IOperation isn't annotated), so this had to change to match the nullable annotations of `ReportDiagnostic` that it's passed to.